test/operators.jl

```diff
@@ -81,6 +81,14 @@ import Base.<
 @test isequal(minmax(TO23094(2), TO23094(1))[1], TO23094(1))
 @test isequal(minmax(TO23094(2), TO23094(1))[2], TO23094(2))
 
+let
+    m = Module()
```

Comment:
don't make anonymous modules unless you must. they unnecessarily complicate the tests

Reply:
I prefer using it because it makes sure we can avoid name collision.
`f42195(...) = ...` as toplevel code would look better ?